database_schemas/cassandra/benchflow.cql

```diff
@@ -23,14 +23,26 @@ CREATE TABLE database_sizes (
   trial_id text,
   dbms text,
   database_name text,
-  database_table text,
   size bigint,
   PRIMARY KEY ((trial_id, experiment_id), dbms)
 );
 
+CREATE TABLE io_data (
```

Comment:
@VincenzoFerme But aren't they cumulative the way they are collected? So it's only one row per trial and container